clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/provider/agent/GoogleLoadBalancerCachingAgent.groovy

```diff
@@ -193,12 +193,16 @@ class GoogleLoadBalancerCachingAgent extends AbstractGoogleCachingAgent implemen
         loadBalancers << newLoadBalancer
 
         if (forwardingRule.target) {
-          def targetPoolName = Utils.getLocalName(forwardingRule.target)
-          def targetPoolsCallback = new TargetPoolCallback(googleLoadBalancer: newLoadBalancer,
-                                                           httpHealthChecksRequest: httpHealthChecksRequest,
-                                                           instanceHealthRequest: instanceHealthRequest)
+          def forwardingRuleTokens = forwardingRule.target.split("/")
 
-          compute.targetPools().get(project, region, targetPoolName).queue(targetPoolsRequest, targetPoolsCallback)
+          if (forwardingRuleTokens[forwardingRuleTokens.size() - 2] != "targetVpnGateways") {
```

Comment:
The url pattern shouldn't change since it is observed by all of our platform resources.
And the `[]` lookup uses `getAt()` under the covers. That method won't throw `ArrayIndexOutOfBoundsExceptions`. It will return `null` if you're off of the end of the collection, and the final token if you're into the negatives.
So feels ok to me.